Deprecate SRID value `variable` (#1937)
oasis-tcs/odata-specs: odata-csdl/1 Introduction.md

[Section ##PrimitiveTypes]| 
Allow stream-valued non-binding parameters| 
[525](https://github.com/oasis-tcs/odata-specs/issues/525)
[Section ##SRID]| 
SRID value `variable` is deprecated| 

> [ 1935](https://github.com/oasis-tcs/odata-specs/issues/1935)

Reviewer: This is not resolved.

Some of these types allow facets, defined in section
"[Type Facets](#TypeFacets)".

> Representation of primitive type values is usage specific, see for example
> - rule `primitiveLiteral` in [OData-ABNF](#ODataABNF) for the
> representation in URLs, and
> - [OData-JSON](#ODataJSON) for the representation in JSON request and
> response bodies.

Author: ```suggestion
Representation of primitive type values within a URL is defined by the rule `primitiveLiteral` in [OData-ABNF](#ODataABNF).
Representation within request and response bodies is format specific.
```

The value of the `SRID` facet MUST be a non-negative integer or the
special value `variable`. If no value is specified, the facet defaults
to `0` for `Geometry` types or `4326` for `Geography` types.

> Services SHOULD NOT use the special value `variable` as, for example, the [OData-JSON](#ODataJSON)
> format does not define a representation for instance-specific spatial reference systems.

Author: ```suggestion
Services SHOULD NOT use the special value `variable` as some formats, for example [OData-JSON](#ODataJSON), do not define a representation for instance-specific spatial reference systems.
```
